build2.py

```diff
@@ -1,223 +0,0 @@
-#!/usr/bin/env python
```

Comment:
so `build.py` has now been replaced with what used to be `build2.py` ?

Reply:
yes, and the old build.py is deleted.